.devcontainer/devcontainer.json

```
@@ -8,50 +8,66 @@
 	// If you want to run the production version of the container, comment out the image and initializeCommand lines above and uncomment the line below.
 	// "image": "public.ecr.aws/w5r9l1c8/swsoc-docker-lambda-base:latest",
 	// "initializeCommand": "docker logout public.ecr.aws && docker pull public.ecr.aws/w5r9l1c8/swsoc-docker-lambda-base:latest",
-	// Set *default* container specific settings.json values on container create.
-	"settings": {
-		"python.pythonPath": "/usr/bin/python3",
-		"python.languageServer": "Pylance",
-		"python.linting.enabled": true,
-		"python.linting.pylintEnabled": true,
-		"python.formatting.blackPath": "/usr/local/bin/black",
-		"python.formatting.provider": "black",
-		"python.formatting.blackArgs": [
-			"--line-length",
-			"100"
-		],
-		"python.testing.unittestEnabled": false,
-		"python.testing.pytestEnabled": true,
-		"python.linting.lintOnSave": true,
-		"python.linting.flake8Enabled": true,
-		"editor.formatOnSave": true,
-		"python.linting.banditPath": "/usr/local/bin/bandit",
-		"python.linting.flake8Path": "/usr/local/bin/flake8",
-		"python.linting.mypyPath": "/usr/local/bin/mypy",
-		"python.linting.pycodestylePath": "/usr/local/bin/pycodestyle",
-		"python.linting.pydocstylePath": "/usr/local/bin/pydocstyle",
-		"python.linting.pylintPath": "/usr/bin/pylint",
-		"terminal.integrated.defaultProfile.linux": "bash (login)",
-		"terminal.integrated.profiles.linux": {
-			"bash (login)": {
-				"path": "bash"
-			}
-		}
+	"customizations": {
+		"vscode": {
+			// Set *default* container specific settings.json values on container create.
+			"settings": {
+				"python.pythonPath": "/usr/bin/python3",
+				"python.testing.unittestEnabled": false,
+				"python.testing.pytestEnabled": true,
+				"python.languageServer": "Pylance",
+				"python.linting.enabled": true,
+				
+				//PyLint Settings
+				"pylint.enabled": true,
+				"pylint.path": "/usr/bin/pylint",
```

```suggestion
				"pylint.path": ["/usr/bin/pylint"],
```
Also needs to be an array:  https://marketplace.visualstudio.com/items?itemName=ms-python.pylint